### ISSUE TITLE
TypeFace GetFamilyNames return String[]?

### ISSUE THREAD

**Issue by AnzerWall** [JetBrains/skija#108]
it's not FontFamilyName[]?

```java
extern "C" JNIEXPORT jobjectArray JNICALL Java_org_jetbrains_skija_Typeface__1nGetFamilyNames
  (JNIEnv* env, jclass jclass, jlong ptr) {
    SkTypeface* instance = reinterpret_cast<SkTypeface*>(static_cast<uintptr_t>(ptr));
    SkTypeface::LocalizedStrings* iter = instance->createFamilyNameIterator();
    std::vector<SkTypeface::LocalizedString> names;
    SkTypeface::LocalizedString name;
    while (iter->next(&name)) {
        names.push_back(name);
    }
    iter->unref();
    jobjectArray res = env->NewObjectArray((jsize) names.size(), skija::FontFamilyName::cls, nullptr);
    for (int i = 0; i < names.size(); ++i) {
        jstring nameStr = javaString(env, names[i].fString);
        jstring langStr = javaString(env, names[i].fLanguage);
        jobject obj = env->NewObject(skija::FontFamilyName::cls, skija::FontFamilyName::ctor, nameStr, langStr);
        env->SetObjectArrayElement(res, i, obj);
    }
    return res;
}

```

```java
public static native Font[] _nGetFamilyNames(long ptr)

```
what?

```java
    public String[] getFamilyNames() {
        try {
            Stats.onNativeCall();
            return _nGetFamilyNames(_ptr);
        } finally {
            Reference.reachabilityFence(this);
        }
    }
```
what?

**Comment by tonsky** [comment]
Probably a mistake. Do you want to provide a patch?

**Comment by tonsky** [comment]
Fixed in 0.91.8. Thank you for reporting!
